tests/test_lab_train.py

```diff
@@ -73,6 +75,35 @@ def mock_convert_llama_to_gguf(model, pad_vocab):
     return LINUX_GGUF_FILE
 
 
+def run_default_phased_train(cli_runner):
+    result = cli_runner.invoke(
+        lab.ilab,
+        [
+            "--config=DEFAULT",
+            "model",
+            "train",
+            "--pipeline",
+            "accelerated",
+            "--strategy",
+            "lab-multiphase",
+            "--phased-phase1-data",
+            "knowledge_data_path",
+            "--phased-phase2-data",
+            "skills_data_path",
+            "--phased-mt-bench-judge",
+            "mt_bench_judge",
+            "--phased-phase1-num-epochs",
+            "1",
+            "--phased-phase2-num-epochs",
+            "1",
+            "--device",
```

Comment:
I don't think you need to pass this, but you could have identified a case in which it's necessary.

Reply:
I either need to pass --device or mock is_high_fidelity for it to make it into the accelerated code block.